docs/docusaurus.config.js

```diff
@@ -82,13 +82,17 @@ const config = {
         name: "load-versions",
         async loadContent() {
           try {
-            const noirVersionPath = path.resolve(
-              __dirname,
-              "../yarn-project/noir-compiler/src/noir-version.json"
+            const noirCompilerPackage = JSON.parse(
+              fs.readFileSync(
+                path.resolve(
+                  __dirname,
+                  "../yarn-project/noir-compiler/package.json"
+                ),
+                "utf-8"
+              )
             );
-            const noirVersion = JSON.parse(
-              fs.readFileSync(noirVersionPath).toString()
-            ).tag;
+            const noirVersion =
+              noirCompilerPackage.dependencies["@noir-lang/noir_wasm"];
```

Comment:
Are the versions of `nargo` and `noir-wasm` the same? Asking because I'm seeing [`v0.18.0-aztec.4`](https://github.com/noir-lang/noir/releases/tag/v0.18.0-aztec.4) as the latest nargo and [`0.18.0-a098541.aztec`](https://www.npmjs.com/package/@noir-lang/noir_wasm/v/0.18.0-a098541.aztec) as the latest on npm. 

If so, then we cannot remove the noir-version json in the compiler and replace it with the wasm dependency, since we still need to tell the user what version of nargo to install if they don't want to use the wasm, right?

Reply:
Yeah, you're right. I'll restore the json and maintain it manually at least until we can get nargo and noir to follow the same version naming scheme